app.py

```diff
@@ -1,19 +1,45 @@
 from flask import Flask, render_template, request, redirect, url_for
 import session_items as session
+import requests as Requests
+from Item import Item
+import os
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
 @app.route('/')
 def index():
-    item_list = session.get_items()
+    item_list = []
+    list_id = '5f56323626c33d81cd98b386'
+    r = Requests.get(f'https://api.trello.com/1/lists/{list_id}/cards?key={os.getenv("KEY")}&token={os.getenv("TOKEN")}')
+    for items in (r.json()):
+        item_list.append( Item(items['id'], 'To Do', items['name']) )
     return render_template("index.html", items=item_list)
 
 @app.route('/newItem', methods=['POST'])
 def submitNewItem():
-    session.add_item(request.form.get('itemName'))
-    item_list = session.get_items()
-    return render_template("index.html", items=item_list)
+    item_name = request.form.get('itemName')
+    list_id = '5f56323626c33d81cd98b386'
```

Comment:
Even though this is not a secret, it could still be made into an environment variable so that it can be varied. (You will need to do so for the selenium test in module 3)

Same for the done list id.